resources/assets/css/_forms.css

```diff
@@ -42,16 +42,17 @@
     @apply text-theme-primary-600;
 }
 
-.input-group:focus-within .input-wrapper-with-prefix,
+.input-group:focus-within .input-wrapper-with-prefix-suffix,
```

Comment:
these now have `-prefix-suffix` but shouldn't that be just `-suffix`? so you get `input-wrapper-with-prefix` and `input-wrapper-with-suffix`